obspy/clients/fdsn/routers/fedcatalog_client.py

```diff
@@ -593,7 +593,8 @@ def get_waveforms_bulk(self, bulk, quality=None, minimumlength=None,
              :meth:`~obspy.fdsn.clients.Client.get_stations_bulk`
 
         >>> client = FederatedClient()
-        >>> bulkreq = "IU ANMO * ?HZ 2010-02-27T06:30:00 2010-02-27T06:33:00"
+        >>> bulkreq = str("IU ANMO * ?HZ 2010-02-27T06:30:00 "
```

Comment:
I hope the `str` is not necessary here? The parantheses alone should be sufficient to allow the line break..